make_release.ps1

```diff
@@ -111,8 +111,8 @@ New-Item -ItemType Directory -Force -Path $libDir | Out-Null
 Copy-Item "$ReleaseDir\c2pa.h" $includeDir -Force
 Copy-Item "$ReleaseDir\c2pa_c.*" $libDir -Force
 
-$zipPath = "$artifactsDir\c2pa-v$version-$Platform.zip"
-Compress-Archive -Path "$includeDir", "$libDir" -DestinationPath $zipPath -Force    
+$zipPath = "$artifactsDir\c2pa-v$version-$platform.zip"
```

Comment:
The variable name used is case-sensitive, and it's sued lower-case above in the script/